packages/wrangler/src/pages/publish.tsx

```diff
@@ -335,6 +345,16 @@ export const Handler = async ({
 		logger.log(`✨ Uploading _redirects`);
 	}
 
+	if (filepathRoutingConfig) {
+		formData.append(
+			"functions-filepath-routing-config.json",
+			new File(
+				[filepathRoutingConfig],
+				"functions-filepath-routing-config.json"
+			)
```

Comment:
can we log we are uploading this file pls? I have found these logs so helpful so many times.
`logger.log(`✨ Uploading functions-filepath-routing-config.json`);

Reply:
I'm not sure it makes since to log this to the user - it's more of an internal implementation detail so that we can display this data in Dash. The file itself is never user-facing.

Reply:
and if you do add it, the test will probably need some updating as well

Reply:
oh ok. I thought this was something the user will care abt. nvm then, carry on 😆